CMSIS/RTOS2/FreeRTOS/Source/cmsis_os2.c

```diff
@@ -534,13 +534,12 @@ osThreadId_t osThreadNew (osThreadFunc_t func, void *argument, const osThreadAtt
     stack = configMINIMAL_STACK_SIZE;
     prio  = (UBaseType_t)osPriorityNormal;
 
-    name = NULL;
     mem  = -1;
 
     if (attr != NULL) {
-      if (attr->name != NULL) {
-        name = attr->name;
-      }
+      /* No need to check if NULL, NULL is OK */
```

Comment:
Currently this comment comes immediately after a NULL check and is deceiving. Add meaningful comment, something like:
`/* Take the name from attributes */`

Reply:
Done